src/index.html

```diff
@@ -15,8 +15,102 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="#home"
+        class="header-logo__link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="moyo logo"
+          class="header-logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="header_link is-active"
+              href="#apple"
+            >
+              apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="header_link"
+              href="#samsung"
+            >
+              samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="header_link"
+              href="#smartphones"
+            >
+              smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="header_link"
+              href="#laptops%26computers"
```

Comment:
The `href` attribute contains `%26` which is the URL encoded form of '&'. While this is technically correct, it might be better to use a simpler identifier for the fragment, such as `#laptops-computers`, to avoid potential issues with URL encoding in different browsers.